internal/components/compute/registry.go

```diff
@@ -0,0 +1,73 @@
+package compute
+
+import (
+	"github.com/mark3labs/mcp-go/mcp"
+)
+
+// Compute-related tool registrations
+
+// RegisterVMSSInfoByNodePoolTool registers the get_vmss_info_by_node_pool tool
+func RegisterVMSSInfoByNodePoolTool() mcp.Tool {
+	return mcp.NewTool(
+		"get_vmss_info_by_node_pool",
+		mcp.WithDescription("Get detailed VMSS configuration for a specific node pool (provides low-level VMSS settings not available in az aks nodepool show)"),
+		mcp.WithString("subscription_id",
+			mcp.Description("Azure Subscription ID"),
+			mcp.Required(),
+		),
+		mcp.WithString("resource_group",
+			mcp.Description("Azure Resource Group containing the AKS cluster"),
+			mcp.Required(),
+		),
+		mcp.WithString("cluster_name",
+			mcp.Description("Name of the AKS cluster"),
+			mcp.Required(),
+		),
+		mcp.WithString("node_pool_name",
+			mcp.Description("Name of the node pool to get VMSS information for"),
```

Comment:
how about combine the following get_all_vmss_by_cluster together here, where empty node_pool_name means all nodepools?